### DISCUSSION TITLE
How can I set a background image in i3?

### DISCUSSION THREAD

**Discussion by jeanbza** [i3/i3#4898] [Q&A] [answered]
I can do `feh --bg-scale /usr/local/home/deklerk/Documents/desert.jpg` to set a background image.

But, adding `exec xset --no-startup-id feh --bg-scale /usr/local/home/deklerk/Documents/desert.jpg` to my `~/.config/i3/config` does not seem to work.

I don't really mind whether I use feh or not: what's the recommended way to set a background image?

**Comment by vincentbernat** [comment, marked as answer]
You seem to have an extra `xset` in your line.

**Reply by jeanbza** [reply]
Dangit. Thank you!

**Reply by fenix011** [reply]
Why does the very exact same line not work for me ? :- (

**Reply by deltatron7543** [reply]
do you have a file in  /usr/local/home/deklerk/Documents/ named desert.jpg?
